--- setup.py
+++ setup.py
@@ -101,8 +101,7 @@
         extra_compile_args["nvcc"] += ["-DWIN32_LEAN_AND_MEAN", "-allow-unsupported-compiler"]
 
     current_dir = pathlib.Path(__file__).parent.resolve()
-    glm_path = osp.join(current_dir, "gsplat", "cuda", "csrc", "third_party", "glm")
-    include_dirs = [glm_path, osp.join(current_dir, "gsplat", "cuda", "include")]
+    include_dirs = [osp.join(current_dir, "gsplat", "cuda", "include")]
 
     extension = CUDAExtension(
         "gsplat.csrc",
